tests/fixtures/mr_insertions/cat-x-mr.json

```diff
@@ -306,78 +306,78 @@
         "measures": {
             "count": {
                 "data": [
-                    1.5,
```

Comment:
These changes are because on the last PR you mentioned that the test was a little hard to interpret because there were so many rows of all 0's. Rather than add a new fixture, I changed this one to have a wider variety of numbers.